paperslicer/pipeline.py

```
@@ -62,3 +84,439 @@ def process(self, pdf_path: str) -> PaperRecord:
+                                    min_area_px=80000,
+                                    skip_full_page=True,
+                                )
+                            else:
+                                exported = exp.export_page_images(pdf_path, outdir=outdir)
+                        for e in exported:
+                            if e.get("type") == "table":
+                                tables_meta.append(e)
+                            else:
+                                figures_meta.append(e)
+                except Exception:
+                    pass
```

Ruff (0.12.2)

`try`-`except`-`pass` detected, consider logging the exception

(S110)